src/transformers/models/resnet/configuration_resnet.py

```diff
@@ -0,0 +1,83 @@
+# coding=utf-8
+# Copyright 2022 Microsoft Research, Inc. and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" ResNet model configuration"""
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+RESNET_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "resnet50": "https://huggingface.co/Francesco/resnet50/blob/main/config.json",
```

Comment:
Will all checkpoints be placed under the `microsoft` organization?

To be updated.

Reply:
microsoft